FreeRTOS/Test/CMock/smp/single_priority_no_timeslice_covg/single_priority_no_timeslice_covg_utest.c

```diff
@@ -413,3 +413,123 @@ void test_task_step_tick_xNextTaskUnblockTime_not_equal( void )
     xTickCount = 0U;
     vTaskStepTick((TickType_t)10U);
 }
+
+/**
+ * @brief AWS_IoT-FreeRTOS_SMP_TC-<TBD>
+ * Two tasks with different priortiy are created in this test.
+ * This test will verify that current core will be requested to yield when resuming
+ * a higher priority task from ISR.
+ *
+ * #define configRUN_MULTIPLE_PRIORITIES                    0
+ * #define configUSE_TIME_SLICING                           0
+ * #define configNUMBER_OF_CORES                            (N > 1)
+ *
+ * This test can be run with FreeRTOS configured for any number of cores
+ * greater than 1 .
+ *
+ * Tasks are created prior to starting the scheduler.
```

Comment:
I think the description could probably be simpler (depending on the test). This portion should describe any setup or pre-conditioning required to take a particular code path. Perhaps we should also include a code snippet of the specific code path we are taking for coverage.

My concern is that when code changes it is difficult to determine which code coverage test needs to be updated to match. This can result in creating unnecessary coverage tests over time.

Reply:
Code could also be changed in the future. In this case, code snippet will need extra effort to maintain.

I would like to use API and comment for tracking. At least when we change the code of an API, we can search for the test cases we may also need to update.

For public API, we can use the API name. We can simply search the API to know which test to update.
test_coverage_<API>_<test description>
For private API, I suggest we depicted it explicitly in the comment. If the private function cover is not the intention of this test, we can skip this section.
For example,
```
/**
 * @brief xTaskResumeFromISR - resume higher priority suspended task
 *
 * Two tasks with different priority are created in this test. This test verifies
 * that current core will be requested to yield when resuming a higher priority task
 * from ISR. The return value of xTaskResumeFromISR indicates yield required for the
 * core calling this API.
 *
 * Private function intend to cover :
 * prvYieldForTask - The core calling this API will be requested to yield for a higher
 *                   priority task
 */
void test_coverage_xTaskResumeFromISR_resume_higher_priority_suspended_task( void )
{
    ....
}
```